modules/openapi-generator/src/main/resources/rust-axum/Cargo.mustache

```diff
@@ -54,8 +54,8 @@ http = "1"
 lazy_static = "1"
 regex = "1"
 serde = { version = "1", features = ["derive"] }
+serde_html_form = "0.2"
 serde_json = { version = "1", features = ["raw_value"] }
-serde_urlencoded = "0.7"
```

Comment:
could you please also explain why we need to replace `serde_urlencoded` with `serde_html_form`?

Reply:
`serde_urlencoded` explicitly doesn't support array query params and will never do so. See https://github.com/nox/serde_urlencoded/issues/52 for example. `serde_html_form` is a fork with that in mind. See item 2 in the PR description.